projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -6,39 +6,54 @@ import { useSelect } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 import { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { isModalSupportedByFlow } from './util';
 import './editor.scss';
 
 export const name = 'launchpad-save-modal';
 
 export const settings = {
 	render: function LaunchpadSaveModal() {
-		const isSaving = useSelect(
+		const isSavingSite = useSelect(
 			select => select( editorStore ).isSavingNonPostEntityChanges(),
 			[]
 		);
-		const prevIsSaving = usePrevious( isSaving );
+		const isSavingPost = useSelect( select => select( editorStore ).isSavingPost(), [] );
+
+		const prevIsSavingSite = usePrevious( isSavingSite );
+		const prevIsSavingPost = usePrevious( isSavingPost );
 		const [ isModalOpen, setIsModalOpen ] = useState( false );
 		const [ dontShowAgain, setDontShowAgain ] = useState( false );
 		const [ isChecked, setIsChecked ] = useState( false );
 
 		const { launchpadScreenOption, siteIntentOption } = window?.Jetpack_LaunchpadSaveModal || {};
-		const isInsideSiteEditor = document.getElementById( 'site-editor' ) !== null;
+		const isInsideSiteOrBlockEditor =
+			document.getElementById( 'site-editor' ) !== null ||
+			document.querySelector( '.block-editor' ) !== null;
```

Comment:
@mattwiebe Wondering if this is the best selector to use to identify the post editor 🤔

Reply:
I am open to suggestions here, I was a bit disappointed there was no unique ID I can use as there is for the site editor